crypto/BRCryptoNetwork.c

```diff
@@ -303,6 +305,17 @@ cryptoNetworkSetHeight (BRCryptoNetwork network,
     network->height = height;
 }
 
+extern uint32_t
+cryptoNetworkGetConfirmationsUntilFinal (BRCryptoNetwork network) {
+    return network->confirmationsUntilFinal;
+}
+
+private_extern void
+cryptoNetworkSetConfirmationsUntilFinal (BRCryptoNetwork network,
+                                         uint32_t confirmationsUntilFinal) {
```

Comment:
Do we want this to be a uint64_t (agreed, no chance the value would ever need that magnitude) just to have all "height" values a uniform size?

Reply:
See CORE-533.